packages/formik/src/Formik.tsx

```diff
@@ -300,26 +294,43 @@ export function useFormik<Values extends FormikValues = FormikValues>({
             prev = setIn(prev, fieldKeysWithValidation[index], curr);
           }
           return prev;
-        }, {})
-      );
+        }, {} as FormikErrors<Values>);
```

Comment:
you should be able to move this type in the `.reduce<FormikErrors<Values>>`
```suggestion
        }, {});
```

Reply:
I need it to be casted in the `.reduce` I think, no?

Reply:
this should give yo the same outcome
```
       fieldErrorsList.reduce<FormikErrors<Values>>((prev, curr, index) => {
          if (curr === 'DO_NOT_DELETE_YOU_WILL_BE_FIRED') {
            return prev;
          }
          if (curr) {
            prev = setIn(prev, fieldKeysWithValidation[index], curr);
          }
          return prev;
        }, {})
```

Reply:
Is does. Generics and casting are essentially the same thing though 🙁 
I made the change anyway